src/main/java/mat/server/humanreadable/cql/CQLHumanReadableGenerator.java

```diff
@@ -3,38 +3,32 @@
 import java.io.IOException;
 import java.util.HashMap;
 import java.util.Map;
-
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 import org.springframework.ui.freemarker.FreeMarkerTemplateUtils;
-
-import freemarker.core.ParseException;
 import freemarker.template.Configuration;
-import freemarker.template.MalformedTemplateNameException;
 import freemarker.template.TemplateException;
-import freemarker.template.TemplateNotFoundException;
 
 @Component
 public class CQLHumanReadableGenerator {
 
-	@Autowired private Configuration freemarkerConfiguration;
+	@Autowired
+    private Configuration freemarkerConfiguration;
```

Comment:
something is wrong with line indentation

Reply:
fixed